app/javascript/components/enrollment/steps/Contact.js

```diff
@@ -44,6 +45,27 @@ class Contact extends React.Component {
     if (event.target.id === 'primary_telephone' || event.target.id === 'secondary_telephone') {
       value = value.replace(/-/g, '');
     }
+    this.updatePrimaryContactMethodValidations(event);
+
+    if (event.target.id === 'primary_telephone' && event.target.value.replace('_', '').length === 12) {
```

Comment:
What is the length check against 12 doing?

Reply:
This is to make sure we're not doing POSTs to checked BlockedNumber if the field isn't even complete

Reply:
Got it, thanks!